cns/cnsclient/cnsclient.go

```diff
@@ -198,3 +198,111 @@ func (cnsClient *CNSClient) DeleteHostNCApipaEndpoint(networkContainerID string)
 
 	return nil
 }
+
+// RequestIPAddress calls the requestIPAddress in CNS
+func (cnsClient *CNSClient) RequestIPAddress(orchestratorContext []byte) (*cns.GetNetworkContainerResponse, error) {
+	var (
+		err      error
+		response *cns.GetNetworkContainerResponse
```

Comment:
requestIPConfigHandler returns GetIPConfigResponse . Isn't this function calling requestipconfighandler?

Reply:
updated